.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
   - "node"
-  - "8"
-  - "6"
+  - "12"
```

Comment:
maybe node js v14 should be added here as its current LTS?

Reply:
@ryanwillis Could you please add this as well?

Reply:
Done.